packages/indentfix/setup.py

```diff
@@ -0,0 +1,38 @@
+import ast
+import io
+import re
+
+from setuptools import setup, find_packages
+
+with io.open('README.md', 'rt', encoding="utf8") as f:
+    readme = f.read()
+
+_description_re = re.compile(r'description\s+=\s+(?P<description>.*)')
+
+with open('lektor_indentfix.py', 'rb') as f:
+    description = str(ast.literal_eval(_description_re.search(
+        f.read().decode('utf-8')).group(1)))
+
+setup(
+    author='maxbachmann',
+    author_email='kontakt@maxbachmann.de',
```

Comment:
in der vorrigend datei standen aber andere Authoren :roll_eyes:

Reply:
die fixed version stammt von mir aber der Großteil ist eben copy paste code von jinja2 daher werden entsprechend auch die ursprünglichen Autoren genannt und die Lizenz beibehalten

Reply:
"  author='maxbachmann'," - das beinhaltet ursprüngliche authoren? interessante definition!